lib/custom_typings/images.d.ts

```diff
@@ -0,0 +1,19 @@
+declare module '*.png' {
+  const value: any;
+  export = value;
+}
+
+declare module '*.jpg' {
+  const value: any;
+  export = value;
+}
+
+declare module '*.svg' {
+  const value: any;
+  export = value;
+}
+
+declare module '*.gif' {
+  const value: any;
+  export = value;
+}
```

Comment:
This part was required for TS to understand that webpack can import image files directly (and that they shouldn't be parsed like regular code files)